conda_smithy/azure_ci_utils.py

```diff
@@ -1,29 +1,28 @@
 import os
 import typing
 
-from vsts.vss_connection import VssConnection
 from msrest.authentication import BasicAuthentication
-
-from vsts.build.v4_1.build_client import BuildClient
-from vsts.build.v4_1.models import BuildDefinitionReference
 from vsts.build.v4_1.build_client import BuildClient
-from vsts.task_agent.v4_0.task_agent_client import TaskAgentClient
-from vsts.task_agent.v4_0.models import TaskAgentQueue
-from vsts.service_endpoint.v4_1.service_endpoint_client import (
-    ServiceEndpointClient
+from vsts.build.v4_1.models import (
+    BuildDefinition,
+    BuildDefinitionReference,
+    SourceRepositories,
+    SourceRepository,
 )
 from vsts.service_endpoint.v4_1.models import ServiceEndpoint
-from vsts.build.v4_1.build_client import BuildClient
-from vsts.build.v4_1.models import SourceRepositories, SourceRepository
-
+from vsts.service_endpoint.v4_1.service_endpoint_client import ServiceEndpointClient
+from vsts.task_agent.v4_0.models import TaskAgentQueue
+from vsts.task_agent.v4_0.task_agent_client import TaskAgentClient
+from vsts.vss_connection import VssConnection
 
+AZURE_ORG_OR_USER = os.getenv("AZURE_ORG_OR_USER", "conda-forge")
+# This will only need to be changed if you need to point to a non-standard azure
+# instance.
 AZURE_TEAM_INSTANCE = os.getenv(
-    "AZURE_INSTANCE", "https://dev.azure.com/conda-forge"
-)
-AZURE_PROJECT_ID = os.getenv("AZURE_PROJECT_ID", "feedstock-builds")
-AZURE_SERVICE_ENDPOINT_NAME = os.getenv(
-    "AZURE_SERVICE_ENDPOINT", "conda-forge"
+    "AZURE_INSTANCE", f"https://dev.azure.com/{AZURE_ORG_OR_USER}"
 )
+AZURE_PROJECT_NAME = os.getenv("AZURE_PROJECT_NAME", "feedstock-builds")
```

Comment:
This large diff is to correct an incorrect name.  `PROJECT_ID` is a UUID associated with an Azure project.